fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordMetaData.java

```diff
@@ -208,6 +214,24 @@ public int getVersion() {
         return version;
     }
 
+    /**
+     * Get value of the counter used for index subspace keys if the counter-based assignment is used.
+     * @return the value of the counter
+     * @see {@link RecordMetaDataBuilder#enableCounterBasedSubspaceKeys()}
+     */
+    public long getSubspaceKeyCounter() {
+        return subspaceKeyCounter;
+    }
+
+    /**
+     * Checks if counter-based subspace keys assignment is used.
+     * @return {@code true} if {@code usesSubspaceKeyCounter} is set
+     * @see {@link RecordMetaDataBuilder#enableCounterBasedSubspaceKeys()}
```

Comment:
```suggestion
     * @see RecordMetaDataBuilder#enableCounterBasedSubspaceKeys()
```

Reply:
all of the `@see`s are already done, I guess, no Idea why your revision doesn't have them.